File: library/src/main/java/cn/cqray/android/dialog/amin/BounceOut.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 弹出
 * @author Cqray
 */
public class BounceOut extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.bounceOut();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/BounceOut.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 弹出
 * @author Cqray
 */
class BounceOut : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.bounceOut()
    }
}